modules/core/src/main/scala/org/tessellation/modules/Services.scala

```diff
@@ -64,5 +67,6 @@ sealed abstract class Services[F[_]] private (
   val metrics: Metrics[F],
   val gossip: Gossip[F],
   val consensus: Consensus[F, GlobalSnapshotEvent, GlobalSnapshotKey, GlobalSnapshotArtifact],
-  val dag: DAGService[F]
+  val dag: DAGService[F],
+  val collateralService: CollateralService[F]
```

Comment:
Can you name it accordingly to the pattern, please? without the Service suffix. The same for trait/class name

Reply:
Changed.